gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/web/internal/configureWebApplication.kt

```diff
@@ -60,6 +58,99 @@ internal fun Project.configureWeb(
     }
 }
 
+private fun Project.introduceComposeWebCompatibilityTask(targets: Collection<KotlinJsIrTarget>) {
+    val webProductionDist = layout.buildDirectory.dir("dist/web/productionExecutable")
+
+    val jsAppRenamed = "__jsApp.js"
+    val wasmAppRenamed = "__wasmApp.js"
+
+    val jsDistTask = targets.firstOrNull { it.name == "js" } ?: return
+    val wasmDistTask = targets.firstOrNull { it.name == "wasmJs" } ?: return
+
+    registerTask<Copy>("composeWebCompatibilityDist") {
+        val jsTargetModuleName = jsDistTask.outputModuleName
+        val wasmTargetModuleName = wasmDistTask.outputModuleName
```

Comment:
Do both `outputModuleName` have some default value when a user doesn't explicitly specify it in the DSL?